internal/fuzzbinder/fuzzscraper.go

```diff
@@ -366,28 +381,48 @@ func GetFuzzTargets(fuzzParameters *FuzzParameters, date string) ([]string, erro
 
 // GetEvidences gets the list of the evidence files used by the fuzzscraper.
 // The expected date format is "YYYYMMDD".
-func GetEvidences(date string, fuzztargets []string, fuzzParameters *FuzzParameters) map[string]string {
-	evidences := make(map[string]string)
+func GetEvidences(date string, fuzzTargets []string, fuzzParameters *FuzzParameters) ([]amber.ClaimEvidence, error) {
+	evidences := make([]amber.ClaimEvidence, len(fuzzTargets)+2)
 	// Get the GCS absolute path of the file containing the revision hash of the source code used
 	// in the coverage build on a given day.
-	evidences["revision"] = fmt.Sprintf("gs://%s/%s/srcmap/%s.json", CoverageBucket, fuzzParameters.ProjectName, date)
+	role := "revision"
+	blobName := fmt.Sprintf("%s/srcmap/%s.json", fuzzParameters.ProjectName, date)
+	uri := fmt.Sprintf("gs://%s/%s", CoverageBucket, blobName)
+	digest, err := getGCSFileDigest(CoverageBucket, blobName)
+	if err != nil {
+		return nil, err
+	}
+	evidences[0] = amber.ClaimEvidence{Role: role, URI: uri, Digest: *digest}
+
 	// Get the GCS absolute path of the file containing the coverage summary for the project on a given day.
-	evidences["project coverage"] = fmt.Sprintf("gs://%s/%s/reports/%s/linux/summary.json", CoverageBucket, fuzzParameters.ProjectName, date)
-	for _, fuzztarget := range fuzztargets {
-		// The role of the coverage evidence using the fuzztarget.
-		role := fmt.Sprintf("%s_%s_%v coverage", fuzzParameters.FuzzEngine, fuzzParameters.ProjectName, fuzztarget)
+	role = "project coverage"
+	blobName = fmt.Sprintf("%s/reports/%s/linux/summary.json", fuzzParameters.ProjectName, date)
+	uri = fmt.Sprintf("gs://%s/%s", CoverageBucket, blobName)
+	digest, err = getGCSFileDigest(CoverageBucket, blobName)
+	if err != nil {
+		return nil, err
+	}
+	evidences[1] = amber.ClaimEvidence{Role: role, URI: uri, Digest: *digest}
+	for idx, fuzzTarget := range fuzzTargets {
+		// The role of the coverage evidence using the fuzzTarget.
+		role := fmt.Sprintf("%s_%s_%v coverage", fuzzParameters.FuzzEngine, fuzzParameters.ProjectName, fuzzTarget)
```

Comment:
Do the roles have to be different?

Also, why is it needed to include the project name in the role? Is not that the same for all targets?
I think all these could have the same role: "Fuzz target". What do you think?

Reply:
The roles here are related to the extracted data. For example, some evidence is for project coverage and some for fuzzer coverage. So I tried to give them different roles depending on their use. My goal here is to help reproduce the code since the files can be very similar.

Reply:
> I think all these could have the same role: "Fuzz target". What do you think?
Each fuzz target has its one coverage summary.

Reply:
> I think all these could have the same role: "Fuzz target". What do you think?

I updated it to "fuzzTarget coverage" to have the same role for all fuzz-targets as discussed.